common/src/main/java/com/wynntils/models/raid/RaidModel.java

```diff
@@ -39,13 +44,17 @@ public class RaidModel extends Model {
     private static final Pattern RAID_COMPLETED_PATTERN = Pattern.compile("§f§lR§#4d4d4dff§laid Completed!");
     private static final Pattern RAID_FAILED_PATTERN = Pattern.compile("§4§kRa§c§lid Failed!");
 
+    private static final Pattern RAID_CHOOSE_BUFF_PATTERN = Pattern.compile("(\\w+) has chosen the (\\w+ \\w+) buff!");
```

Comment:
This pattern is a bit too weak, although unlikely if someone sent a regular message of "player has chosen the X buff!" this will catch it. You should make it expect the raid chat banner and also keep the colours, as far as I'm aware all raids use the same colours for the buff messages so no need to ignore those.

Reply:
Noted, I can see how someone could trigger it by sending such a message. The reason I went the route of stripping the colours as you've mentionned in other comments is that it felt like features that rely on colours could become a chore to update if wynn just happens to decide they like pink more than orange for example. Happy to change it back though as originally I did have it this way.

Reply:
It's not too much of a bother. Wynn changes various things that we have to adapt for, so having to change the colour in a pattern is at least an easy fix

Reply:
Should be resolved now. I've removed the color Util file that I added and made the pattern a complete match with what chat says when gaining a buff (either hex code for others or yellow for yourself)

Reply:
Using original message removes the yellow (or other colour possibilities as i found out) as a problem